Model name : r2attunet
Training Model : R2Att-Unet | status : [42mNEW
[41mFinished Initalization
[41mStart Training
  0%|          | 0/200 [00:00<?, ?it/s]
  0%|          | 0/200 [00:02<?, ?it/s]]
Traceback (most recent call last):
  File "./run.py", line 12, in <module>
    main()
  File "./run.py", line 9, in main
    trainer.fit()
  File "/home/oj/spark/train/train.py", line 351, in fit
    preds = self.model(images)
  File "/home/oj/anaconda3/envs/fire/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/oj/anaconda3/envs/fire/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1520, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/oj/spark/train/../network/models.py", line 464, in forward
    d2 = self.Up_RRCNN2(d2)
  File "/home/oj/anaconda3/envs/fire/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/oj/anaconda3/envs/fire/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1520, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/oj/spark/train/../network/models.py", line 134, in forward
    x1 = self.RCNN(x)
  File "/home/oj/anaconda3/envs/fire/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/oj/anaconda3/envs/fire/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1520, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/oj/anaconda3/envs/fire/lib/python3.8/site-packages/torch/nn/modules/container.py", line 217, in forward
    input = module(input)
  File "/home/oj/anaconda3/envs/fire/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/oj/anaconda3/envs/fire/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1520, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/oj/spark/train/../network/models.py", line 120, in forward
    x1 = self.conv(x+x1)
  File "/home/oj/anaconda3/envs/fire/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/oj/anaconda3/envs/fire/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1520, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/oj/anaconda3/envs/fire/lib/python3.8/site-packages/torch/nn/modules/container.py", line 217, in forward
    input = module(input)
  File "/home/oj/anaconda3/envs/fire/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/oj/anaconda3/envs/fire/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1520, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/oj/anaconda3/envs/fire/lib/python3.8/site-packages/torch/nn/modules/conv.py", line 460, in forward
    return self._conv_forward(input, self.weight, self.bias)
  File "/home/oj/anaconda3/envs/fire/lib/python3.8/site-packages/torch/nn/modules/conv.py", line 456, in _conv_forward
    return F.conv2d(input, weight, bias, self.stride,
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 96.00 MiB. GPU 0 has a total capacity of 9.77 GiB of which 92.69 MiB is free. Including non-PyTorch memory, this process has 9.68 GiB memory in use. Of the allocated memory 7.94 GiB is allocated by PyTorch, and 368.92 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)